src/views/HomePageView.vue

```diff
@@ -58,7 +71,7 @@ import LicitatiiComponent from "@/components/LicitatiiComponent.vue";
 @Component({
   components: { LicitatiiComponent }
 })
-export default class App extends Vue {
+export default class HomePageView extends Vue {
```

Comment:
@azirbrooks This should have been caught at code review!

Reply:
@DoruNow Failing to catch that is related to my limited experience with Vue